data/mods/BrightNights/bionics.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "id": "bio_solar",
+    "type": "bionic",
+    "name": "Solar Panels",
+    "description": "Installed on your back is a set of retractable solar panels.  When in direct sunlight, they will automatically deploy and slowly recharge your power level.",
```

Comment:
You decided to make them automatically deploy, not retract and fold by will?

Reply:
Requires less atention from the player, and also had waves why they wont get damaged when fighting. Seems convenient all around.